backend/app/controllers/api/v1/demo_controller.rb

```
@@ -0,0 +1,100 @@
+      message: "🚨 DEMO MODE - Authentication bypassed",
+      dividend_rounds: rounds.map { |round| dividend_round_json(round, include_dividends: true) },
+      pagination: {
+        current_page: 1,
+        per_page: 25,
+        total_pages: 1,
+        total_count: rounds.count
+      }
+    }
+  end
+  
+  private
```

RuboCop (1.75.5)

[convention] Remove a blank line after `private`.

(Layout/EmptyLinesAroundAccessModifier)